scipy/optimize/tests/test_linprog.py

```diff
@@ -359,6 +358,48 @@ def f(c, A_ub=None, b_ub=None, A_eq=None, b_eq=None, bounds=None):
 
         assert_raises(ValueError, f, [1, 2], A_ub=np.zeros((1, 1, 3)), b_eq=1)
 
+    def test_bounds_fixed(self):
+
+        def g(c, bounds):
+            res = linprog(c, bounds=bounds, method=self.method, options=self.options)
+            return res
```

Comment:
It looks like you might have put this in a function to avoid the lines getting too long?
Consider [parameterizing the test](https://docs.pytest.org/en/latest/parametrize.html). I think it will accomplish the same thing and make it more concise. Maybe also check the number of iterations to ensure that this is solved in presolve?

Reply:
Actually I just copied the style of the neighboring tests. But I see that these function definitions are only used to check is errors or warnings are raised. I'll stick to that.

Reply:
And I'll include a check on `res.nit == 0`.